framework/db/pgsql/QueryBuilder.php

```diff
@@ -67,6 +67,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
         Schema::TYPE_BINARY => 'bytea',
         Schema::TYPE_BOOLEAN => 'boolean',
         Schema::TYPE_MONEY => 'numeric(19,4)',
+        Schema::TYPE_JSON => 'json',
```

Comment:
It's better to use `Schema::TYPE_JSON => 'jsonb'`
https://www.postgresql.org/docs/current/static/datatype-json.html